training/trillium/Mixtral-8x7B-Pytorch/XPK/benchmark.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+source env.sh
```

Comment:
Oh, I forgot, this will be under `/app/env.sh` because XPK create a new docker for us.